addon/mixins/application-route-mixin.js

```diff
@@ -60,11 +60,11 @@ export default Ember.Mixin.create({
   /**
     This action is triggered whenever the session is successfully
     authenticated. If there is a transition that was previously intercepted
-    by
-    [`AuthenticatedRouteMixin#beforeModel`](#SimpleAuth-AuthenticatedRouteMixin-beforeModel)
+    by 
+    {{#crossLink "Mixins.AuthenticatedRouteMixin/beforeModel:method"}}AuthenticatedRouteMixin#beforeModel{{/crossLink}}
```

Comment:
True but it was required because you were using `@namespace Mixins` in your yuidoc markup for your mixin classes. We can remove the `@namespace Mixins`.